docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,417 @@
+:::
+
+## Checking the Integration
+
+To check that the UID2 module has a valid UID2 token, call `pbjs.getUserIds().uid2`. If a value is returned, a valid UID2 token still exists in the UID2 module.
+
+If there are problems with the integration, here are some steps you can take:
+
+- Check the browser console logs.
+- Use the browser developer tools to inspect the API calls to the UID2 service.
+
+For additional help, refer to Prebid's documentation on [Troubleshooting Prebid.js](https://docs.prebid.org/troubleshooting/troubleshooting-guide.html) and [Debugging Prebid.js](https://docs.prebid.org/debugging/debugging.html).
+
+An example of a tool for validating and debugging Prebid.js configuration is Professor Prebid, an open-source Chrome extension:
+
+- Chrome web store download location: [Professor Prebid](https://chromewebstore.google.com/detail/professor-prebid/kdnllijdimhbledmfdbljampcdphcbdc)
+- Documentation on prebid.org: [Professor Prebid User Guide](https://docs.prebid.org/tools/professor-prebid.html)
+
+## Configuration Parameters for userSync
+
+The following parameters apply only to the UID2 Prebid User ID Module integration.
+
+(**GWH_SW question I missed asking you. When we say "Optional, client refresh" -- what does that mean? Is it required for client refresh? Or N/A for server-only and optional for client refresh? And will readers know? If something is required for a specific mode we should say that.**)
+
+| Param under userSync.userIds[] | Scope | Type | Description | Example |
+| --- | --- | --- | --- | --- |
+| name | Required | String | ID value for the UID2 module. Always `"uid2"`. | `"uid2"` |
+| value | Optional, server-only mode | Object | An object containing the value for the advertising token. | See [Configuration Parameter Examples: Value](#configuration-parameter-examples-value). |
+| params.uid2Token | Optional, client refresh mode | Object | The initial UID2 token. This should be the `body` element of the decrypted response from a call to the `/token/generate` or `/token/refresh` endpoint. | See [Sample Token](#sample-token). |
+| params.uid2Cookie | Optional, client refresh mode | String | The name of a cookie that holds the initial UID2 token, set by the server. The cookie should contain JSON in the same format as the uid2Token param. **If uid2Token is supplied, this param is ignored.** | See [Sample Token](#sample-token). |
+| params.uid2ApiBase | Optional, client refresh mode | String | Overrides the default UID2 API endpoint. | `"https://prod.uidapi.com"` (the default)|
+| params.storage | Optional, client refresh mode | String | Specify the module internal storage method: `cookie` or `localStorage`. We recommend that you do not provide this parameter. Instead, allow the module to use the default. | `localStorage` (the default) |
```

Comment:
probably need @lionell-pack-ttd to confirm if params.storage is really Only Optional for client-refresh mode and it's not even optional for server-only mode?

Reply:
@sunnywu : also noted in source doc as an additional query.

Reply:
(this was resolved by LP who confirmed that params.storage is optional for both.)